proto/security.proto

```diff
@@ -0,0 +1,48 @@
+// messages related to the oberon protocol
+// See: https://github.com/mikelodder7/oberon
+syntax = "proto3";
+
+package okapi.security;
+
+option csharp_namespace = "Okapi.Security";
+option go_package = "github.com/trinsic-id/okapi";
+option java_package = "trinsic.okapi";
+
+
+// Create a new oberon token
+message CreateOberonTokenRequest {
+    bytes sk = 1; // raw BLS key bytes
+    bytes data = 2; // data is the public part of the oberon protocol and can be any data
+    repeated bytes blinding = 3; // optional blinding for the token
```

Comment:
I would separate the blinding process from token creation.
We want the holder of token to be able to blind the token with their own secret.
A blind and unblind endpoints that literally just to - and + operations on the token would be fine.

Reply:
I added blind/unblind functionality.

I left the ability to provide blinds in the original token creation call since they are optional and there are use cases where you would want to create a blinded token in one pass.